src/pages/index.js

```diff
@@ -63,27 +100,102 @@ function handleImageClick(cardLink, cardCaption) {
   imagePopup.open(cardLink, cardCaption);
 }
 
+function handleAvatarClick() {
+  editAvatarPopup.open();
+}
+
+function handleDeletePopup(cardId, evt) {
+  deleteVerifyPopup.open();
+  cardIdForDeletion = cardId;
+  cardToDelete = evt.target.closest(".card");
+}
+
+function removeLikeFromServer(cardId, likeButton) {
+  api
+    .removeLikeFromCard(cardId)
+    .then(likeButton.classList.remove("card__liked_active"))
```

Comment:
In blocks `then`, `catch`, `finally` you need to pass only the **reference** to the function (not a function call) otherwise the function is called immediately without waiting for the response from the server. You can pass an **unnamed** arrow function as well.